docs/source/crds.rst

```diff
@@ -0,0 +1,118 @@
+.. _custom-resources-used-by-the-test-operator:
+
+==========================================
+Custom Resources Used By the test-operator
+==========================================
+
+This file contains definitions of all custom resources that are accepted by
+the test-operator. Currently, there are two custom resouces:
+
+* :ref:`tempest-custom-resource`
+
+* :ref:`tobiko-custom-resource`
+
+
+.. _tempest-custom-resource:
+
+Tempest Custom Resource
+=======================
+
+.. literalinclude:: ../../config/samples/test_v1beta1_tempest.yaml
+   :language: yaml
+
+
+.. _custom-tempest-configuration:
+
+Custom Tempest Configuration
+----------------------------
+To configure tempest via tempest.conf use the :code:`tempestconfRun.overrides`
+parameter. This parameter accepts a list of key value pairs that specify values
+that should be written to tempest.conf generated inside the container.
+
+For example this definition of Tempest object:
+
+.. code-block:: yaml
+
+    ---
+    apiVersion: test.openstack.org/v1beta1
+    kind: Tempest
+    metadata:
+      name: tempest-tests
+      namespace: openstack
+    spec:
+      containerImage: quay.io/podified-antelope-centos9/openstack-tempest:current-podified
+      tempestRun:
+        includeList: | # <-- Use | to preserve \n
+          tempest.api.identity.v3.*
+        concurrency: 8
+      tempestconfRun:
+          overrides: |
+            auth.admin_username admin
+            auth.admin_password 1234
+
+will ensure that tempest will be executed with tempest.conf that looks like this:
```

Comment:
i know i'm being to literal, but a documentation should be as clear as possible. The tempest.conf will not look like that. I'd loose the wording 'looks like this' because it doesn't, it's much longer :D  Also, I find it a bit weird that we focus on tempestconf config while the object above configures also tempest. I'd either remove this part (lines 53-62) or would be more specific.